app-office/texmacs/texmacs-1.99.12.recipe

```diff
@@ -0,0 +1,111 @@
+SUMMARY="A scientific editing platform designed to create beautiful technical documents"
+DESCRIPTION="TeXmacs provides a unified and user friendly framework for editing \
+structured documents with different types of content: text, mathematics, graphics, \
+interactive content, slides, etc.\
+Its rendering engine uses high-quality typesetting algorithms so as to produce \
+professionally looking documents, which can either be printed out or presented from \
+a laptop."
+HOMEPAGE="http://www.texmacs.org"
+COPYRIGHT="Free Software Foundation"
+LICENSE="GNU GPL v3"
+
+SOURCE_URI="https://github.com/mgubi/texmacs/archive/texmacs-$portVersion-haiku-qt5.tar.gz"
+CHECKSUM_SHA256="607fa4db4676b78780f7af086feec2e2332dc18a371d79fa50d640de6db10b43"
+SOURCE_DIR="texmacs-texmacs-$portVersion-haiku-qt5/src"
+PATCHES="texmacs-$portVersion.patchset"
+
+REVISION="1"
+
+ADDITIONAL_FILES="texmacs.rdef.in"
+DISABLE_SOURCE_PACKAGE="yes"
```

Comment:
It is GPL, AFAIK we should generate source package for it.

Reply:
Ok, sorry, I will remove this.